src/events/SDL_pen_c.h

```diff
@@ -24,7 +24,6 @@
 #ifndef SDL_pen_c_h_
 #define SDL_pen_c_h_
 
-#include "../../include/SDL3/SDL_pen.h"
```

Comment:
I actually noticed this a few months ago, it's the only relative path inclusion that exits out of `src` and enters `include`. It caused some problems with a port of SDL to a different build system, so I hope you don't mind me sneaking this in.